### PR TITLE
feat: Log NFT explore events

### PR DESCRIPTION
**Pull request by gbugyis**
- Adds page view and trending row selection events

### CODE REVIEW

File: src/nft/components/explore/Table.tsx [hunk 1]
```diff
@@ -94,21 +96,29 @@ export function Table<D extends Record<string, unknown>>({
           prepareRow(row)
 
           return (
-            <tr
-              className={styles.tr}
-              {...row.getRowProps()}
+            <TraceEvent
+              events={[Event.onClick]}
+              name={EventName.NFT_TRENDING_ROW_SELECTED}
+              properties={{ collection_address: row.original.collection.address }}
```
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/nft/components/explore/Table.tsx:102]: We're only launching on main so this isn't really necessary unless we expand to more chains

**Review by cbachmeier — APPROVED** [review]
LGTM

File: src/nft/components/explore/Table.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gbugyis) [reply src/nft/components/explore/Table.tsx]: We've been adding chainId already in a few places (wherever collection_address is passed), I'll add here as well for consistency.

**Review by lynnshaoyu — APPROVED** [review]

**Comment by lynnshaoyu** [comment]
> 

in response to charlie's comments about the chain, more chains to come apparently soon, so wna retain historical information about which chain the first nft events are coming from

**Pull request merged**